articles/references/domain-designer/associations/associations.md

```diff
@@ -42,100 +40,100 @@ This can be achieved by setting the association like this:
 
 ### Aggregation Association
 
->Aggregation relationship
+> Aggregation relationship
 >
->1. When representing a software or database relationship, e.g. car model engine ENG01 is part of a car model CM01, as the engine, ENG01, may be also part of a different car model.[8]
->2. When the container is destroyed, the contents are usually not destroyed, e.g. a professor has students; when the professor dies the students do not die along with them.
-
-*[Wikipedia - Composition](https://en.wikipedia.org/wiki/Class_diagram#Composition)*
+> 1. When representing a software or database relationship, e.g. car model engine ENG01 is part of a car model CM01, as the engine, ENG01, may be also part of a different car model.[8]
+> 2. When the container is destroyed, the contents are usually not destroyed, e.g. a professor has students; when the professor dies the students do not die along with them.
+>
+> *- [Wikipedia](https://en.wikipedia.org/wiki/Class_diagram#Composition)*
 
-![Aggregation Association](images/association-aggregate-professor.png)
+![How an Aggregation Association appears in UML and Intent Architect's Domain Designer](images/association-aggregate-professor.png)
 
-This can be achieved by setting the association like this:
+An Aggregation Association be achieved by setting the properties of the Association as follows:
 
 |Source End|Target End|
 |-|-|
 |⬜ Navigable|✅ Navigable|
```

Comment:
Not really important, but it's annoying this blue checkbox:

![image](https://user-images.githubusercontent.com/1290262/128498778-d7815651-9723-456d-966c-507b45c57404.png)

Doesn't seem to render in colour with the font that DocFX uses.

Reply:
I wanted to make use of some visual symbols to indicate which properties should be checked or not and the grey box and green checkbox seemed to be of equal size so that it would keep everything aligned. Can't remember if I tried the blue one.

Reply:
BTW, any idea how this would look like on an OS other than Windows?

Reply:
Edge and Firefox on Ubuntu:

![image](https://user-images.githubusercontent.com/1290262/128509491-591571d8-4f40-4a10-832c-bb2ec121ea0c.png)

Edge on macOS:

![image](https://user-images.githubusercontent.com/1290262/128509582-46329e01-9352-4aac-8987-846a967cceff.png)

Safari on macOS:

![image](https://user-images.githubusercontent.com/1290262/128509619-164db035-522c-4625-9bc9-683ebd2413a3.png)